bin/run_data_task.py

```diff
@@ -60,18 +60,18 @@ def main():
         )
```

Comment:
Note that I've removed the executable bit from `run_data_task.py`. Ruff complains if a file is marked as executable that doesn't have a shebang line. It always seems to be run as `python bin/run_data_task.py` so the executable bit isn't needed